src/GregClient/Requests/Hosts.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using RestSharp;
+
+namespace Greg.Requests
+{
+    public class Hosts : Request
```

Comment:
it may not be standard practice in this repo -but in other Dynamo repos we try to add /// summary tags for public methods/classes - if some of these are inherited from the base class it's okay to skip unless theres something specific.

Reply:
especially in open source repos.

Reply:
No idea why the comments look so weird ...

Reply:
spaces vs tabs